### PR TITLE
Made scripts for the docker scripts

### PR DESCRIPTION
**Pull request by kariharju**
Needs linux chmod and test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,16 @@ This project installs and configures Jenkins for you in a Docker container, with
...
-    * Windows: ```docker run -p 8080:8080 --env JAVA_OPTS=-Dhudson.model.DirectoryBrowserSupport.CSP= -v jenkins_home://var/jenkins_home -v //var/run/docker.sock:/var/run/docker.sock --detach --name robodemo robodemo```
-4. Open browser to http://localhost:8080
-5. Login with admin / RoboCorpRules2020!
-6. Click "robodemo" to open the demo project
-7. Click Scan multibranch pipeline from the left-hand side
-8. Refresh, see that the job starts scanning first, then building the master branch
-9. Click to open the master, check the results. Click build #1 from left-hand side to see full console output and more information.
+1. Setup the docker
+   * Windows users: `get-started.bat`
+   * Linux/Mac: `chmod 755 get-started.sh` ->  `./get-started.sh`
```
Comment (jyrmyx) [inline README.md]: chmod not needed anymore?

**Review by jyrmyx — APPROVED** [review]
lgtm!

File: README.md (hunk 1 above)
Comment (kariharju) [reply README.md]: I'm always worried if git clients lose the permissions.

Comment (kariharju) [reply README.md]: ..put then again if they are lost linux user probably know what to do.

**Pull request merged**